skin-schema.json

```diff
@@ -507,6 +515,14 @@
               "iOS SDK": "Unsupported",
               "Android SDK": "v4.21.0+"
             }
+          },
+          "inactiveColor" : {
```

Comment:
Please make the naming scheme consistent.  So this should be inactiveAccentColor